website/common/script/ops/buy/buyMysterySet.js

```diff
@@ -25,7 +25,10 @@ export default function buyMysterySet (user, req = {}, analytics) {
   }
 
   each(mysterySet.items, item => {
-    user.items.gear.owned[item.key] = true;
+    user.items.gear.owned = {
```

Comment:
could this be taken outside of the loop?

Reply:
done

Reply:
thanks @Xaz16 ! Although it seems the commit is missing

Reply:
Really strange, but I see the same situation, it looks like something wrong with diff view. If you set changes like on screenshot you will see that change...

![image](https://user-images.githubusercontent.com/12300576/71559933-b46f7e00-2a74-11ea-93d3-b998ee4d7051.png)

Reply:
I think there's just one place left https://github.com/HabitRPG/habitica/pull/11637/files#diff-cc435769f01f8a6e9f26679a4f82c09fR127

Reply:
Done @paglias